exporter/awsxrayexporter/doc.go

```diff
@@ -14,4 +14,6 @@
 
 // Package awsxrayexporter implements an OpenTelemetry Collector exporter that sends trace data to
 // AWS X-Ray in the region the collector is running in using the PutTraceSegments API.
+
+//go:generate mdatagen metadata.yaml
```

Comment:
Nit: the godoc is usually left above the package definition and the go:generate line is above that doc string

Reply:
intellij auto formatting annoyingly disagrees. But I have forced it to do as i say!

Reply:
Weird! I wonder if there's a golint rule for this